automation/capitanodoc/utils.ts

```diff
@@ -126,9 +126,7 @@ export class MarkdownFileParser {
 				} else {
 					reject(
 						new Error(
-							`Markdown section not found: title="${title}" file="${
-								this.mdFilePath
-							}"`,
+							`Markdown section not found: title="${title}" file="${this.mdFilePath}"`,
```

Comment:
There are now 3 commits in this PR. One of the commits runs `prettier` (which I had hoped to avoid) in order to fix a build error: [Uncommitted changes found (^^^ diff above ^^^) - FAIL](https://travis-ci.org/balena-io/balena-cli/jobs/562531248)